lib/debounce.py

```diff
@@ -1,22 +1,27 @@
 # Taken from https://gist.github.com/walkermatt/2871026
 
-
 from threading import Timer
 
 
 def debounce(wait):
-    """ Decorator that will postpone a functions
+  """ Decorator that will postpone a functions
```

Comment:
This is not properly formatted, which is likely why it's not being line-wrapped correctly.  See e.g. https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
this is a direct copy+paste from github FYI, which is why it is a bit odd